src/geocel/GeantGeoUtils.cc

```diff
@@ -191,6 +214,10 @@ void reset_geant_geometry()
 #if G4VERSION_NUMBER >= 1100
         G4ReflectionFactory::Instance()->Clean();
 #endif
+        free_and_clear(G4Material::GetMaterialTable());
+        free_and_clear(G4Element::GetElementTable());
+        free_and_clear(const_cast<std::vector<G4Isotope*>*>(
+            G4Isotope::GetIsotopeTable()));
```

Comment:
@drbenmorgan This is an inconsistency between the Geant4 isotope store and the material/element stores. Not sure if we should open a PR to fix it...